Sources/TuistGenerator/Generator/InfoPlistContentProvider.swift

```diff
@@ -96,7 +96,7 @@ final class InfoPlistContentProvider: InfoPlistContentProviding {
         var packageType: String?
 
         switch target.product {
-        case .app:
+        case .app, .appClip:
```

Comment:
I'd add a test to make sure that we are returning the right `CFBundlePackageType` when the target is an app clip.